src/validators.ts

```diff
@@ -30,7 +30,10 @@ export const maxValue = (value: number): ValidationRule =>
 
 export const required = <ValidationRule>withI18nMessage(_required)
 
-export const address = <ValidationRule>withI18nMessage(isAddress)
+export const address = <ValidationRule>withI18nMessage(value => {
+  if (!value) return true
+  return isAddress(value)
+})
```

Comment:
Just to reduce amount of code lines.

```ts
export const address = <ValidationRule>withI18nMessage(value => !value ? true : isAddress(value))
```

but you can leave it as it is if you want, nothing wrong with that, just the matter of preference